setup.py

```diff
@@ -18,30 +18,29 @@
     url='https://github.com/department-of-veterans-affairs/notification-utils',
     license='MIT',
     author='Department of Veteran Affairs',
+    # author_email is required if the author is specified.
+    author_email='unspecified',
     description='Shared python code for VA Notify. Forked from https://github.com/cds-snc/notification-utils',
     long_description=__doc__,
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
         'bleach>=4.1.0',
-        'mistune==0.8.4',
-        'requests>=2.26.0',
-        'idna>=3.3',
-        'python-json-logger>=0.1.11',
+        'boto3>=1.24.61',
+        'botocore>=1.27.61',
         'Flask>=1.1.1',
-        'orderedset>=2.0.3',
-        'Jinja2>=2.11.3',
-        'statsd>=3.3.0',
         'Flask-Redis>=0.4.0',
-        'pyyaml==5.4.1',
-        'phonenumbers==8.12.12',
-        'pytz>=2021.3',
-        'smartypants>=2.0.1',
+        'Jinja2>=2.11.3',
+        'mistune==0.8.4',
         'monotonic>=1.6',
+        'orderedset>=2.0.3',
+        'phonenumbers~=8.12.12',
```

Comment:
phonenumbers is a requirement here but not in tests.  are there no tests related to the phonenumbers library?

Reply:
This repo doesn't not have any tests for "phonenumbers", but the "phonenumbers" package [has its own tests](https://github.com/daviddrysdale/python-phonenumbers/tree/dev/python/tests).  If we added unit tests in this repo, making it a requirement in requirements_for_test.txt would not be necessary because those requirements are in addition to those in setup.py, which includes "phonenumbers".